### ISSUE TITLE
Gutenberg support planned ?

### ISSUE THREAD

**Issue by bricedupuy** [TheCraigHewitt/Seriously-Simple-Podcasting#288]
Is Gutenberg support planned ?  
If so what kind of blocks can we expect ?

**Comment by jonathanbossenger** [comment]
Hi @bricedupuy thanks for the question.

At the moment the Seriously Simple Podcasting plugin does not expose any editor UI, and therefore works with Gutenberg out of the box.

Once Gutenberg is merged into core I'm sure we'll consider creating custom blocks to replace any short code specific functionality.
